packages/snowpack/src/commands/build.ts

```diff
@@ -61,6 +62,28 @@ export async function command(commandOptions: CommandOptions) {
   const {cwd, config} = commandOptions;
   const messageBus = new EventEmitter();
 
+  // TODO: update this with a more robust / typed log event
+  function logEvt({level, id, msg, args}) {
+    let logger = console.log;
+    let color = (stdout: string) => stdout;
+    if (level === 'warn') {
+      logger = console.warn;
+      color = colors.yellow;
+    }
+    if (level === 'error') {
+      color = colors.red;
+      logger = console.error;
+    }
+    const output = msg || util.format.apply(util, args);
+    logger(`${id ? colors.blue(`[${id}] `) : ''}${color(output)}`);
+  }
+  messageBus.on('CONSOLE', logEvt);
+  messageBus.on('WORKER_COMPLETE', logEvt);
+  messageBus.on('WORKER_MSG', logEvt);
+  messageBus.on('WORKER_RESET', logEvt);
+  messageBus.on('WORKER_START', logEvt);
+  messageBus.on('WORKER_UPDATE', logEvt);
```

Comment:
I think there’s a solution where we think through a better logging solution throughout the entire app, but this seemed like the lowest-effort solution.

I think others have suggested a `--debug` flag which I’m +1000 for. But implementing that should be more of a logging rewrite IMO.